src/shared/services/api/api/flexiRecords.js

```
@@ -503,6 +503,336 @@ export async function multiUpdateFlexiRecord(sectionid, scouts, value, column, f
+ * @param {string|boolean} [options.dob='1'] - Include date of birth field (1/0/true/false)
+ * @param {string|boolean} [options.age='1'] - Include age field (1/0/true/false)
+ * @param {string|boolean} [options.patrol='1'] - Include patrol field (1/0/true/false)
+ * @param {string} [options.type='none'] - Record type
+ * @returns {Promise<Object|null>} Creation result with flexirecordid
+ * @throws {Error} When API request fails or validation errors
+ * 
+ * @example
+ * const result = await createFlexiRecord(123, 'Viking Section Movers', token);
+ * console.log(`Created FlexiRecord with ID: ${result.flexirecordid}`);
+ */
+export async function createFlexiRecord(sectionid, name, token, options = {}) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'createFlexiRecord'. Consider to delete it or rename it.

'createFlexiRecord' is defined here:


(lint/suspicious/noRedeclare)